bin/jruby-truffle

```diff
@@ -0,0 +1 @@
+truffleruby
```

Comment:
We can probably get rid of this one if `rbconfig.rb` is updated (and a grep in the code base).